src/main/java/Pokemon.java

```diff
@@ -0,0 +1,57 @@
+public abstract class Pokemon {
+    private String name;
+    private int level;
+    private String pokemonType;
+
+//    constructor
+    public Pokemon(String name, int level){
+        this.name=name;
+        this.level=level;
+//        this.pokemonType=pokemonType;
+    }
+
+//    methode 1
+    public void levelUp(){
```

Comment:
Deze methode wordt in elke subklasse overschreven, misschien is het daarom handig om deze abstract te maken?